app/models/event_stream.rb

```diff
@@ -35,14 +36,22 @@ class EventStream < ApplicationRecord
 
   after_commit :emit_notifications, :on => :create
 
+  supports_not :timeline, :reason => "Only subclasses can be on a timeline"
```

Comment:
For example, this reason will never be presented in the UI nor API, which goes against why you'd need it.

Reply:
Yeah, that's more for developers.  I know supports is a provider concept but it describes what I wanted to do.  

I originally had `self.timeline_supported? => false` in the EventStream class and only set `self.timeline_supported? => true` for `MiqEvent` and `EmsEvent` classes.

I'm not against using something else, but saw it and thought I was reinventing the wheel.

Reply:
Changed to respond_to? in the timeline_options method.